plugin.py

```diff
@@ -12,25 +12,18 @@
 from LSP.plugin.core.views import range_to_region  # TODO: not public API :(
 import sublime
 
-VERSION = "1.38.2"
-URL = "https://github.com/OmniSharp/omnisharp-roslyn/releases/download/v{}/omnisharp-{}.zip"  # noqa: E501
+VERSION = "1.39.10"
+URL = "https://github.com/OmniSharp/omnisharp-roslyn/releases/download/v{}/omnisharp-{}-net6.0.zip"  # noqa: E501
```

Comment:
Does this require users to have .NET 6 installed?

Reply:
In Linux, It does.

Reply:
No I think it's fine to just write in the README.md to tell users to make sure they have .NET 6 installed.